dapps/onboarding.vue/src/components/signed-in/signed-in.ts

```diff
@@ -26,6 +26,7 @@ import { Prop } from 'vue-property-decorator';
 import { EvanComponent } from '@evan.network/ui-vue-core';
 import * as bcc from '@evan.network/api-blockchain-core';
 import * as dappBrowser from '@evan.network/ui-dapp-browser';
+import { bccHelper, session, lightwallet } from '@evan.network/ui-session';
```

Comment:
what are all the imports used for? The component is empty... or is this just a mock component to be filled later?